src/controllers/service_data.ts

```diff
@@ -0,0 +1,105 @@
+import ApiManagementClient from "azure-arm-apimanagement";
+import { fromOption, toError } from "fp-ts/lib/Either";
+import { Option } from "fp-ts/lib/Option";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseSuccessJson,
+  ResponseErrorForbiddenNotAuthorized,
+  ResponseErrorInternal,
+  ResponseErrorNotFound,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { OrganizationFiscalCode } from "italia-ts-commons/lib/strings";
+
+import {
+  getApimUser,
+  IExtendedUserContract,
+  isAdminUser
+} from "../apim_operations";
+import * as config from "../config";
+import { getApimAccountEmail, SessionUser } from "../utils/session";
+
+import {
+  fromEither,
+  fromPredicate,
+  TaskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import nodeFetch from "node-fetch";
+
+// tslint:disable-next-line:no-any
+type ServiceDataSuccessResponse = any /* TODO: shape as the expected response from the API */;
```

Comment:
This should be the shape of the response expected from the API

Reply:
Fix with https://github.com/pagopa/io-developer-portal-backend/pull/184/commits/4d72bcc3efdc00892ffb6eff5bd96a0189066854